packages/cli/package.json

```diff
@@ -62,6 +62,7 @@
     "@types/node": "^10",
     "@types/web3": "^1.0.18",
     "globby": "^8",
+    "prettier": "1.13.5",
```

Comment:
I believe this shouldn't be necessary since it's already in the root package json.
Ditto for the typescript line just below it. @mcortesi can you confirm? I believe you had a PR to remove these from the package.json files a while back?

Reply:
When I ran `yarn run docs`, it complained that `prettier` was an unknown command, so I had to add it to run it